MMUL-BASH-01/Exercises/Day-2/Exercise.007.solutions

```diff
@@ -0,0 +1,41 @@
+1) Supposing your user is "mmul" and your machine is "machine":
+
+> ssh mmul@machine
+
+2) Query the status of the service:
+
+sudo systemctl status mmul_exercise.service
```

Comment:
Please change using $(whoami) instead of mmul

Reply:
replaced username with $(whoami) command